doc/dev/configuration.rst

```diff
@@ -0,0 +1,9 @@
+.. _`configuration`:
+
+Configuration
+=============
+
+Chaosinventory uses Pythons `ConfigParser <https://docs.python.org/3/library/configparser.html>`_ for user configuration. The used configuration file may be loaded from different directories.
+
+Chaosinventory always loads the :code:`src/chaosinventory/fallback.cfg` first, which initalizes the empty sections. Every use of configuration data must work with only the fallback data available.
```

Comment:
As said in the other comment, this file should not be added as it's out of scope and not needed